### PR TITLE
[camera] Implement lockCaptureOrientation

### PR DESCRIPTION
**Pull request by bbrto21**
Signed-off-by: Boram Bae <boram21.bae@samsung.com>

### CODE REVIEW

File: packages/camera/tizen/src/camera_plugin.cc [hunk 1]
```diff
@@ -241,9 +246,22 @@ class CameraPlugin : public flutter::Plugin {
...
+        std::string orientation;
+        OrientationType orientation_type;
+        if (GetValueFromEncodableMap(arguments, "orientation", orientation) &&
+            StringToOrientationType(orientation, orientation_type)) {
+          camera_->LockCaptureOrientation(orientation_type);
+          result->Success();
+          return;
+        }
+        result->Error("InvalidArguments", "Please check 'orientation'");
+      }
```
Comment (bwikbs) [inline packages/camera/tizen/src/camera_plugin.cc:261]: How about putting some logs here?  I mean for error case handling

Comment (bwikbs) [reply packages/camera/tizen/src/camera_plugin.cc:261]: It's just a suggestion... I guess you know the reason :imp:

**Comment by bwikbs** [comment]
Thank you for your dedicated work! It's pretty awesome :+1:

File: packages/camera/tizen/src/camera_plugin.cc (hunk 1 above)
Comment (bbrto21) [reply packages/camera/tizen/src/camera_plugin.cc:261]: Well... I want to reduce unnecessary logs as much as possible. in this case, throwing an error directly to ```flutter``` and leaving enough logs in the wrapper implementation. so I think it is enough

Comment (bwikbs) [reply packages/camera/tizen/src/camera_plugin.cc:261]: Got it.

**Review by bwikbs — APPROVED** [review]
LGTM

**Pull request merged**